src/liballoc/vec.rs

```diff
@@ -2397,6 +2398,21 @@ impl<T: Clone> From<&mut [T]> for Vec<T> {
     }
 }
 
+#[stable(feature = "vec_from_array", since = "1.44.0")]
+impl<T, const N: usize> From<[T; N]> for Vec<T>
+where
+    [T; N]: LengthAtMost32,
+{
```

Comment:
Any docs for this?

Reply:
Does it need documentation? It looks pretty self-explanatory to me.